components/brave_ads/core/internal/account/deposits/deposits_database_table.cc

```diff
@@ -43,8 +44,17 @@ size_t BindColumns(const mojom::DBActionInfoPtr& mojom_db_action,
 
   size_t row_count = 0;
 
+  std::set<std::string> creative_instance_ids;
```

Comment:
maybe flat_set

Reply:
As these will not be large, good point. Changed. Thanks